package.json

```diff
@@ -25,7 +25,7 @@
 	},
 	"dependencies": {
 		"@sapphire/discord-utilities": "^2.1.5",
-		"@sapphire/discord.js-utilities": "next",
+		"@sapphire/discord.js-utilities": "^3.0.0-next.e5418d0d.0",
```

Comment:
I just noticed I changed it from `next` to a specific next version. Is it ok?

Reply:
No, change it to `next`. That's important otherwisee yarn/npm will install 2 versions when installing framework causing conflicts.